Framework/DataObjects/src/EventList.cpp

```diff
@@ -28,6 +28,7 @@
 #include <cmath>
 #include <functional>
 #include <limits>
+#include <numeric>
```

Comment:
I don't see why this library was included, did you actually use something from this? I am curious which other libraries listed here are not needed.

Reply:
It was for an intermediate version of the code and has been removed